Core.ApplicationServices/Model/KitosEvents/KitosEventMapper.cs

```diff
@@ -6,8 +6,8 @@ public class KitosEventMapper : IKitosEventMapper
 {
     public KitosEventDTO MapKitosEventToDTO(KitosEvent kitosEvent)
     {
-        var message = JsonConvert.SerializeObject(kitosEvent.EventBody.ToKeyValuePairs());
-        var topic = kitosEvent.Topic;
-        return new KitosEventDTO(message, topic);
+        var payload = kitosEvent.EventBody;
+        var topic = kitosEvent.Topic + kitosEvent.EventBody.Type();
```

Comment:
since eventBody is extracted in the previous line, the "Type" method can be called on the payload